DVC file -> .dvc file (2nd chunk) reopen (#1408)
iterative/dvc.org: content/docs/command-reference/status.md

                  [targets [targets ...]]

positional arguments:

>   targets        Limit command scope to these stages or `.dvc` files.
>                  Using -R, directories can also be given, to search
>                  `.dvc` files in.

Reviewer: ```suggestion
  targets        Limit command scope to these stages or .dvc files.
                 Using -R, directories can also be given, to search
                 .dvc files in.
```

| remote | `--remote`     | Comparisons are made between the cache, and the given remote. Remote storage is defined using the `dvc remote` command.     |
| remote | `--cloud`      | Comparisons are made between the cache, and the default remote, typically defined with `dvc remote --default`.              |

DVC determines which data and code files to compare by analyzing all stages (in
[`dvc.yaml`](/doc/user-guide/dvc-file-format)) and
[`.dvc` files](/doc/user-guide/dvc-file-format) in the <abbr>workspace</abbr>
(the `--all-branches` and `--all-tags` options compare multiple workspace
versions).

> The comparison can be limited to certain stages (in `dvc.yaml`) and `.dvc` files

Reviewer: ```suggestion
The comparison can be limited to certain stages (in `dvc.yaml`) or `.dvc` files
```

only, by listing them as `targets`. (Changes are reported only against these.)
When this is combined with the `--with-deps` option, a search is made for
changes in other stages that affect each target.

> In the `local` mode, changes are detected through the hash value of every file

Reviewer: ```suggestion
In the local mode, changes are detected through the hash value of every file
```

> listed in every `.dvc` file in question against the corresponding file in the

Reviewer: Lets do this here:
"listed in every stage (in `dvc.yaml` or `.dvc` files) in..."

| remote | `--cloud`      | Comparisons are made between the cache, and the default remote, typically defined with `dvc remote --default`.              |

DVC determines which data and code files to compare by analyzing all

> [`.dvc` files](/doc/user-guide/dvc-file-format) in the <abbr>workspace</abbr>

Reviewer: analyzing all stages (in `dvc.yaml`) and `.dvc` files

Reviewer: This change applies all over this doc. Please try to generalize this in this file. Status checks stages (in dvc.yaml files) as well as .dvc files. Thanks

Reviewer: I checked status quickly and I don't see any more mentions of dvc.yaml. Please try to use "dvc.yaml" AND/OR ".dvc file(s)" as it seems appropriate in every comparison.

Reviewer: And request my review when ready. Thanks

Reviewer: Still no changes to status... I'll make a specific review.

be executed by `dvc repro`.

If instead, differences are detected, `dvc status` lists those changes. For each

> `.dvc` file (stage) with differences, the changes in <abbr>dependencies</abbr>

Reviewer: Simplify here to
```suggestion
stage with differences, the changes in <abbr>dependencies</abbr>
```

and/or <abbr>outputs</abbr> that differ are listed. For each item listed, either
the file name or hash is shown, and additionally a status word is shown
describing the changes (described below).

> - _changed checksum_ means that the <abbr>`.dvc` file</abbr> hash has changed

Reviewer: ```suggestion
- _changed checksum_ means that the stage i(n dvc.yaml) or .dvc file hash has changed
```

Reviewer: p needs wrapping

Reviewer: Oops thanks for fixing `i(n`

  (e.g. someone manually edited the file).

- _always changed_ means that this is a `.dvc` file with no dependencies (an

>   _orphan_ stage file) or that it has the `always_changed: true` value set (see
>   `--always-changed` option in `dvc run`), so its considered always changed, and
>   thus is always executed by `dvc repro`.

Reviewer: ```suggestion
  _orphan_ stage file) or that the stage in `dvc.yaml` has the `always_changed: true` value set (see
  `--always-changed` option in `dvc run`).
```
> p needs wrapping

Reviewer: Missed removing the whole last sentence. I'll open a new comment on this.

- _changed deps_ or _changed outs_ means that there are changes in dependencies

>   or outputs tracked by the <abbr>`.dvc` file</abbr>. Depending on the use case,

Reviewer: ```suggestion
  or outputs tracked by the stage in `dvc.yaml` or `.dvc` file. Depending on the use case,
```
> needs wrapping

  commands like `dvc commit`, `dvc repro`, or `dvc run` can be used to update
  the file. Possible states are:

  - _new_: An <abbr>output</abbr> is found in the workspace, but there is no

>     corresponding file hash saved in a `.dvc` file yet.

Reviewer: ```suggestion
    corresponding file hash saved in the `dvc.lock` or `.dvc` file yet.
```
Please link `dvc.lock` to the same user guide for now.

  - _modified_: An output or <abbr>dependency</abbr> is found in the workspace,

>     but the corresponding file hash the `.dvc` file is not up to date.

Reviewer: but the corresponding file hash the `dvc.lock` or `.dvc` file

Author: I changed this to:

> but the corresponding file hash in the `dvc.lock` or `.dvc` file

Is that okay?

Reviewer: Yes thanks I didn't write it correctly. I think we have an extra "the" now though, I'll open a separate comment 🙂

>   - _deleted_: The output or dependency is references in a `.dvc` file, but does

Reviewer: ```suggestion
  - _deleted_: The output or dependency is referenced in a `dvc.yaml` or `.dvc` file, but does
```
needs wrapping

    not exist in the workspace.

>   - _not in cache_: An output exists in workspace and the corresponding file

Reviewer: ```suggestion
  - _not in cache_: An output exists in the workspace, and the corresponding file
```

>     hash in the `.dvc` file is up to date, but there is no corresponding

Reviewer: ```suggestion
    hash in the `dvc.lock` or `.dvc` file is up to date, but there is no corresponding
```

<abbr>outputs</abbr> that differ are listed. For each item listed, either the
file name or hash is shown, and additionally a status word is shown describing
the changes (described below).

- _changed checksum_ means that the stage (in `dvc.yaml`) or `.dvc` file hash
  has changed (e.g. someone manually edited the file).

- _always changed_ means that this is a `.dvc` file with no dependencies (an
  _orphan stage_ (see [`dvc add`](/doc/command-reference/add)) or that the stage
  in `dvc.yaml` has the `always_changed: true` value set (see `--always-changed`
  option in `dvc run`).

- _changed deps_ or _changed outs_ means that there are changes in dependencies
  or outputs tracked by the stage in `dvc.yaml` or `.dvc` file. Depending on the
  use case, commands like `dvc commit`, `dvc repro`, or `dvc run` can be used to
  update the file. Possible states are:

  - _new_: An <abbr>output</abbr> is found in the workspace, but there is no
    corresponding file hash saved in the
    [`dvc.lock`](/doc/user-guide/dvc-file-format) or
    [`.dvc`](/doc/user-guide/dvc-file-format) file yet.
  - _modified_: An output or <abbr>dependency</abbr> is found in the workspace,

>     but the corresponding file hash in the `dvc.lock` or `.dvc` file is not up to
>     date.

Reviewer: Oops fixing formatting
```suggestion
    but the corresponding file hash in the `dvc.lock` or `.dvc` file is not up
    to date.
```

be executed by `dvc repro`.

If instead, differences are detected, `dvc status` lists those changes. For each
stage with differences, the changes in <abbr>dependencies</abbr> and/or
<abbr>outputs</abbr> that differ are listed. For each item listed, either the
file name or hash is shown, and additionally a status word is shown describing
the changes (described below).

- _changed checksum_ means that the stage (in `dvc.yaml`) or `.dvc` file hash
  has changed (e.g. someone manually edited the file).

- _always changed_ means that this is a `.dvc` file with no dependencies (an
  _orphan stage_ (see [`dvc add`](/doc/command-reference/add)) or that the stage
  in `dvc.yaml` has the `always_changed: true` value set (see `--always-changed`

>   option in `dvc run`), so its considered always changed, and thus is always
>   executed by `dvc repro`.

Reviewer: ```suggestion
  option in `dvc run`).
```

- _changed deps_ or _changed outs_ means that there are changes in dependencies
  or outputs tracked by the stage in `dvc.yaml` or `.dvc` file. Depending on the
  use case, commands like `dvc commit`, `dvc repro`, or `dvc run` can be used to
  update the file. Possible states are:

  - _new_: An <abbr>output</abbr> is found in the workspace, but there is no
    corresponding file hash saved in the
    [`dvc.lock`](/doc/user-guide/dvc-file-format) or
    [`.dvc`](/doc/user-guide/dvc-file-format) file yet.
  - _modified_: An output or <abbr>dependency</abbr> is found in the workspace,

>     but the corresponding file hash in the `dvc.lock` or the `.dvc` file is not
>     up to date.

Reviewer: ```suggestion
    but the corresponding file hash in the `dvc.lock` or `.dvc` file is not up to
    date.
```
